model/content_entity.go

```diff
@@ -32,9 +32,85 @@ func (w *ContentEntity) IsValid() bool {
 			return false
 		}
 	}
+
+	EnumValid := w.IsEnumValid()
+
+	if EnumValid == false {
+		return false
+	}
+	return true
+}
+
+//IsEnumValid this loops over the properties of an entity, if the enum is not nil, it will validate if the option the user set is valid
+//If nullable == true, this means "null" can be used as an option in the enumeration
+//If statements are structured around this ^ and covers different cases.
+func (w *ContentEntity) IsEnumValid() bool {
+	for k, property := range w.Schema.Properties {
+		nullFound := false
+		enumFound := false
+		enumOptions := EnumString(property.Value.Enum)
+
+		if property.Value.Enum != nil {
+			enumProperty := w.GetString(k)
+
+			////This checks if a "null" option was provided which is needed if nullable == true
+			if strings.Contains(enumOptions, "null") {
```

Comment:
I don't think we should allow them to actually pass in the string null when making the API call (it's needed as an option for the OpenAPI config but they shouldn't actually pass in "null" calling the endpoint because that is actually a string and other system will recognize it as such